recipes/grafana.rb

```diff
@@ -100,6 +100,18 @@
   mode 0700
 end
 
+directory "#{node['grafana']['base_dir']}/conf/provisioning/dashboards/kserving" do
```

Comment:
Can you call the directory "kserve" not "kserving"? Both this and the remote directory copied in.